File: Week_04/122/best_time_to_buy_and_sell_stock_ii.go
package main

func maxProfit(prices []int) int {
	count := 0
	for i := 1; i < len(prices);i++ {
		if prices[i] > prices[i-1] {
			count += prices[i] - prices[i-1]
		}
	}
	return count
}

File: Week_04/122/best_time_to_buy_and_sell_stock_ii.py
from typing import List


class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        count = 0

        for i in range(1, len(prices)):
            if prices[i] > prices[i - 1]:
                count += prices[i] - prices[i - 1]
        return count
